crates/cairo-lint-core/tests/test_files/bool_comparison/bool_comparison

```diff
@@ -0,0 +1,109 @@
+//! > Comparison with false
```

Comment:
add tests where the bool literal is on the lhs as well. And try to add a test like that:
```rs
fn main() {
    let x = true;
    // This is a comment
    if x == false {
        println!("x is false");
    }
}
```